src/components/EntityMap.vue

```diff
@@ -7,8 +7,8 @@
     @load="onMapLoaded"
   >
     <EntityMapMarker
-      v-for="(coordinates, index) in coordinatesList"
-      :key="`${coordinates[2]}-${coordinates[3]}-${index}`"
+      v-for="(coordinates, idx) in coordinatesList"
+      :key="getMarkerKey(coordinates[2], idx)"
```

Comment:
hm...  I did almost exactly they same thing but I had just put a `selected` computed property and pretty sure I was calculating it the same as you are in this function but I wasn't getting it to work.  But I forgot exactly the combination I had had too.  Glad this works!